xdp-dump/tests/test-xdpdump.sh

```diff
@@ -110,13 +114,10 @@ test_capt_pcap()
     fi
 }
 
-version_greater_or_equal()
-{
-    printf '%s\n%s\n' "$2" "$1" | sort -V -C
-}
-
 test_capt_pcapng()
 {
+    assert_kernel_version_greater_or_equal 5.7.0
```

Comment:
You should be able to make this test pass with the older kernel, guess only lines 137,139,145,175 will probably need to be conditional.